### DISCUSSION TITLE
Mac OSX 15.1.1 failed fetch

### DISCUSSION THREAD

**Discussion by Expalmi** [dani-garcia/vaultwarden#5229] [Q&A]
I do have the latest version of vaultwarden installed.

it works fine with IOS clients and windows clients.

It does not with my the Mac OSX clients 2024.11.2 and before. It get the error failed to fetch.

Deinstall and new install did not fix the problem. Where can I look to solve the problem?

Andreas

**Comment by BlackDex** [comment]
With macOs client you mean the desktop client?
Or safari extension?

**Reply by Expalmi** [reply]
Both, the desktop client and safari  extension

**Comment by mgarces** [comment]
I've ran repeatedly into this issue; "solution" is to uninstall, wipe user data, install and connect again; after a few days, same thing happens.
